src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -11,6 +12,7 @@
 import org.hibernate.Hibernate;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.lang.Nullable;
```

Comment:
we already use `import javax.annotation.Nullable;`
I am not sure what the difference is, but might be good to have it consistent.

Reply:
I was not aware of that! I don't see a difference, so I'll change it.